### DISCUSSION TITLE
Request/Response Size alterations

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1046] [Q&A]
I had an issue with a job failing because the "Request Entity is too large", but after upping some server configurations, it works as expected.  Is there any negatives to increasing any of the response/request configs in the ./waved command? If so, what are those?

![image](https://user-images.githubusercontent.com/78372056/136063024-e047d8be-f675-446a-8af1-5acb0b9a7578.png)

**Comment by lo5** [comment]
The request payload sizes are limited for security reasons. You can increase those limits if your server is deployed on some internal network. 

That said, typically if your app is sending large payloads (5MB+), it implies that payload is ending up on a single webpage somewhere (e.g. a 50MB request is routed to a web browser, which has to render all that information), which could affect browser performance negatively. Displaying large amounts of information might lead to UX issues (scrolling large tables, etc.), which is better solved by limiting the amount of information via pagination, etc.

**Comment by Alex-Wenner-FHR** [comment]
Gotcha, thank you!

**Reply by Alex-Wenner-FHR** [reply]
Just a quick follow up, pagination is not yet an available feature of the tables correct? I know I had mentioned it a while back but not sure if it has been made available!
